Validation/src/Validation/_file.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, filepath, colmod = None, hist_kwargs = dict(), **open_kwargs)
         self.__file = uproot.open(filepath, **open_kwargs)
         self.__colmod = colmod
         self.__df = None
+        self.path = filepath
```

Comment:
Instead of storing our own copy of the filepath, we could just grab uproot's

```python
@property
def path(self):
    return self.__file.file_path
```
would enable the same `f.path` syntax via the `property` decorator.

Reply:
I was thinking there probably was a smarter way to do this :)

Reply:
Something like this?
https://github.com/LDMX-Software/ldmx-sw/pull/1223/commits/13b008ca9723fab73d6b84a869c713524649a490

Reply:
Exactly :+1: